### ISSUE TITLE
Having trouble with requirements.txt

### ISSUE THREAD

**Issue by yipy0005** [Saoge123/PocketFlow#5]
Hi,

Thank you so much for publishing the code, it must have been a lot of work writing the code. I am very keen to try out the repository, but I'm having troubles with installing the `requirements.txt` file because most of the packages seem to be local to the computer that the repository was developed.

Could you provide a clean conda env file via `conda env export --no-builds`?

Also, could you let me know what is the CUDA driver that works with the repository?

Thank you so much and I look forward to your reply. 😁

Best regards,
Yew Mun

**Comment by Saoge123** [comment]
name: PocketFlow
channels:
  - conda-forge
  - schrodinger
  - defaults
dependencies:
  - _libgcc_mutex=0.1
  - _openmp_mutex=5.1
  - alabaster=0.7.12
  - apbs=1.5
  - asttokens=2.0.5
  - babel=2.11.0
  - backcall=0.2.0
  - blas=1.0
  - boost=1.70.0
  - boost-cpp=1.70.0
  - brotlipy=0.7.0
  - bzip2=1.0.8
  - c-ares=1.19.0
  - ca-certificates=2023.7.22
  - cairo=1.16.0
  - certifi=2023.7.22
  - cffi=1.15.1
  - charset-normalizer=2.0.4
  - colorama=0.4.6
  - comm=0.1.2
  - cryptography=39.0.1
  - curl=7.88.1
  - dbus=1.13.18
  - debugpy=1.5.1
  - decorator=5.1.1
  - docutils=0.18.1
  - executing=0.8.3
  - expat=2.4.9
  - fontconfig=2.14.1
  - freemol=1.158
  - freetype=2.12.1
  - giflib=5.2.1
  - glew=2.0.0
  - glib=2.69.1
  - gst-plugins-base=1.14.1
  - gstreamer=1.14.1
  - h5py=3.7.0
  - hdf4=4.2.13
  - hdf5=1.10.6
  - icu=58.2
  - idna=3.4
  - imagesize=1.4.1
  - importlib_metadata=6.0.0
  - intel-openmp=2023.1.0
  - ipykernel=6.19.2
  - ipython=8.12.0
  - ipywidgets=8.0.7
  - jedi=0.18.1
  - jinja2=3.1.2
  - jpeg=9e
  - jupyter_client=8.1.0
  - jupyter_core=5.3.0
  - jupyterlab_widgets=3.0.8
  - krb5=1.20.1
  - lcms2=2.12
  - ld_impl_linux-64=2.38
  - lerc=3.0
  - libclang=10.0.1
  - libcurl=7.88.1
  - libdeflate=1.17
  - libedit=3.1.20221030
  - libev=4.33
  - libevent=2.1.12
  - libffi=3.4.4
  - libgcc-ng=11.2.0
  - libgfortran-ng=7.5.0
  - libgfortran4=7.5.0
  - libglu=9.0.0
  - libgomp=11.2.0
  - libholoplaycore=0.1.0_rc4
  - libllvm10=10.0.1
  - libnetcdf=4.8.1
  - libnghttp2=1.52.0
  - libopenblas=0.3.18
  - libpng=1.6.39
  - libpq=12.15
  - libsodium=1.0.18
  - libssh2=1.10.0
  - libstdcxx-ng=11.2.0
  - libtiff=4.5.0
  - libuuid=1.41.5
  - libwebp=1.2.4
  - libwebp-base=1.2.4
  - libxcb=1.15
  - libxkbcommon=1.0.1
  - libxml2=2.9.14
  - libxslt=1.1.35
  - libzip=1.8.0
  - lz4-c=1.9.4
  - matplotlib-inline=0.1.6
  - mengine=1
  - mkl=2023.1.0
  - mkl-service=2.4.0
  - mkl_fft=1.3.6
  - mkl_random=1.2.2
  - mpeg_encode=1
  - mtz2ccp4_px=1.0
  - ncurses=6.4
  - nest-asyncio=1.5.6
  - nspr=4.35
  - nss=3.89.1
  - openbabel=3.1.1
  - openssl=3.0.10
  - packaging=23.0
  - pandas=1.4.2
  - parso=0.8.3
  - pcre=8.45
  - pdb2pqr=2.1.2+pymol
  - pexpect=4.8.0
  - pickleshare=0.7.5
  - pillow=9.4.0
  - pixman=0.40.0
  - platformdirs=2.5.2
  - ply=3.11
  - pmw=2.0.1+3
  - prompt-toolkit=3.0.36
  - ptyprocess=0.7.0
  - pure_eval=0.2.2
  - pycairo=1.21.0
  - pycparser=2.21
  - pygments=2.15.1
  - pymol=2.4.1
  - pyopenssl=23.0.0
  - pyqt=5.15.7
  - pyqt5-sip=12.11.0
  - pysocks=1.7.1
  - python=3.8.12
  - python-dateutil=2.8.2
  - python-lmdb=1.3.0
  - python_abi=3.8
  - pytz=2023.3
  - pyzmq=25.1.0
  - qt-main=5.15.2
  - qt-webengine=5.15.9
  - qtwebkit=5.212
  - rdkit=2019.09.3
  - readline=8.2
  - rigimol=1.3
  - setuptools=67.8.0
  - sip=6.6.2
  - six=1.16.0
  - snowballstemmer=2.2.0
  - sphinx=5.0.2
  - sphinx_rtd_theme=1.1.1
  - sphinxcontrib-applehelp=1.0.2
  - sphinxcontrib-devhelp=1.0.2
  - sphinxcontrib-htmlhelp=2.0.0
  - sphinxcontrib-jsmath=1.0.1
  - sphinxcontrib-qthelp=1.0.3
  - sphinxcontrib-serializinghtml=1.1.5
  - sqlite=3.41.2
  - stack_data=0.2.0
  - tbb=2021.8.0
  - tk=8.6.12
  - toml=0.10.2
  - tornado=6.2
  - traitlets=5.7.1
  - typing_extensions=4.6.3
  - urllib3=1.26.16
  - wcwidth=0.2.5
  - wheel=0.38.4
  - widgetsnbextension=4.0.8
  - xz=5.2.10
  - zeromq=4.3.4
  - zlib=1.2.13
  - zstd=1.5.5
  - pip:
    - absl-py==1.4.0
    - anyio==4.0.0
    - appdirs==1.4.4
    - argon2-cffi==23.1.0
    - argon2-cffi-bindings==21.2.0
    - arrow==1.3.0
    - async-lru==2.0.4
    - attrs==23.1.0
    - beautifulsoup4==4.12.2
    - bleach==6.1.0
    - cachetools==5.3.1
    - click==8.1.7
    - contourpy==1.1.0
    - cycler==0.11.0
    - defusedxml==0.7.1
    - docker-pycreds==0.4.0
    - easydict==1.10
    - et-xmlfile==1.1.0
    - exceptiongroup==1.1.3
    - fastjsonschema==2.18.1
    - fonttools==4.40.0
    - fqdn==1.5.1
    - gitdb==4.0.11
    - gitpython==3.1.40
    - google-auth==2.21.0
    - google-auth-oauthlib==1.0.0
    - grpcio==1.56.0
    - imageio==2.33.1
    - importlib-metadata==6.7.0
    - importlib-resources==5.12.0
    - ipython-genutils==0.2.0
    - isoduration==20.11.0
    - jax==0.4.13
    - jaxlib==0.4.13
    - joblib==1.2.0
    - json5==0.9.14
    - jsonpointer==2.4
    - jsonschema==4.19.2
    - jsonschema-specifications==2023.7.1
    - jupyter==1.0.0
    - jupyter-console==6.6.3
    - jupyter-events==0.8.0
    - jupyter-lsp==2.2.0
    - jupyter-server==2.9.1
    - jupyter-server-terminals==0.4.4
    - jupyterlab==4.0.7
    - jupyterlab-pygments==0.2.2
    - jupyterlab-server==2.25.0
    - kiwisolver==1.4.4
    - markdown==3.4.3
    - markupsafe==2.1.3
    - matplotlib==3.7.1
    - meeko==0.4.0
    - mistune==3.0.2
    - ml-dtypes==0.2.0
    - msgpack==1.0.7
    - nbclient==0.8.0
    - nbconvert==7.10.0
    - nbformat==5.9.2
    - notebook==7.0.6
    - notebook-shim==0.2.3
    - numpy==1.23.0
    - nvidia-cublas-cu11==11.11.3.6
    - nvidia-cufft-cu11==10.9.0.58
    - oauthlib==3.2.2
    - openpyxl==3.1.2
    - opt-einsum==3.3.0
    - overrides==7.4.0
    - pandocfilters==1.5.0
    - pip==23.3.2
    - pkgutil-resolve-name==1.3.10
    - prometheus-client==0.18.0
    - protobuf==4.23.3
    - psutil==5.9.5
    - pyasn1==0.5.0
    - pyasn1-modules==0.3.0
    - pyg-lib==0.2.0+pt113cu117
    - pyparsing==3.1.0
    - python-json-logger==2.0.7
    - pyyaml==6.0
    - qtconsole==5.4.4
    - qtpy==2.4.1
    - referencing==0.30.2
    - requests==2.31.0
    - requests-oauthlib==1.3.1
    - rfc3339-validator==0.1.4
    - rfc3986-validator==0.1.1
    - rpds-py==0.10.6
    - rsa==4.9
    - scikit-learn==1.2.2
    - scipy==1.10.1
    - seaborn==0.12.2
    - send2trash==1.8.2
    - sentry-sdk==1.39.1
    - setproctitle==1.3.3
    - smmap==5.0.1
    - sniffio==1.3.0
    - soupsieve==2.5
    - tensorboard==2.13.0
    - tensorboard-data-server==0.7.1
    - terminado==0.17.1
    - threadpoolctl==3.1.0
    - tinycss2==1.2.1
    - tomli==2.0.1
    - torch==1.13.0+cu117
    - torch-cluster==1.6.1+pt113cu117
    - torch-geometric==2.3.1
    - torch-scatter==2.1.1+pt113cu117
    - torch-sparse==0.6.17+pt113cu117
    - torch-spline-conv==1.2.2+pt113cu117
    - torchaudio==0.13.0+cu117
    - torchdiffeq==0.2.3
    - torchvision==0.14.0+cu117
    - tqdm==4.65.0
    - types-python-dateutil==2.8.19.14
    - uri-template==1.3.0
    - vina==1.2.5
    - wandb==0.16.1
    - webcolors==1.13
    - webencodings==0.5.1
    - websocket-client==1.6.4
    - werkzeug==2.3.6
    - xgboost==1.0.2
    - zipp==3.15.0
    - zuko==1.0.1

**Comment by Saoge123** [comment]
Hi, thanks for your attention. The code runs on cuda11.3 or 11.7.

**Comment by yipy0005** [comment]
Hi, I have managed to get the repository to work. In addition, I have submitted a pull request to replace `requirements.txt` with `environment.yml` instead. I have also updated the `README.md` to provide instructions on how to create the conda environment. 😁

Cheers!
Yew Mun
